### PR TITLE
Fixes #32601 - Applicability regen after upload and remove

### PR DESCRIPTION
**Pull request by rverdile**
To test:

1) Register a content host
2) Create empty repo
3) Subscribe content host to product with empty repo
4) Install walrus-0.71 from [https://jlsherrill.fedorapeople.org/fake-repos/needed-errata/](https://jlsherrill.fedorapeople.org/fake-repos/needed-errata/) on content host
5) Add walrus-5.21 from [https://jlsherrill.fedorapeople.org/fake-repos/needed-errata/](https://jlsherrill.fedorapeople.org/fake-repos/needed-errata/) to empty repo
	- walrus-5.21 will be shown as an upgradable package on the content host (bug fix - did not previously update automatically)
6) Remove walrus-5.21 from repo
	- will no longer be shown as an upgradable package on the content host (bug fix - did not previously update automatically)

### CODE REVIEW

File: app/lib/actions/katello/repository/upload_files.rb [hunk 1]
```diff
@@ -30,6 +33,7 @@ def plan(repository, files, content_type = nil)
 
             plan_action(FinishUpload, repository, content_type: content_type, upload_actions: upload_actions)
             plan_self(tmp_files: tmp_files)
+            plan_action(Actions::Katello::Applicability::Repository::Regenerate, :repo_ids => [repository.id]) if generate_applicability
           end
         ensure
```
Comment (jlsherrill) [inline app/lib/actions/katello/repository/upload_files.rb:38]: There's actually two ways to upload content.  The first takes the entire file and uploads it (i believe thats this action).  The problem with that is that only files of fairly small sizes are accepted.  This is what the UI uses.

The cli uses a much more complicated series of steps which breaks the file into chunks and uploads each of them and then finally triggers an import.  `app/lib/actions/katello/repository/import_upload.rb`  is what handles that. 

You can test this by running the 'hammer repository upload' command

File: app/lib/actions/katello/repository/remove_content.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@ class RemoveContent < Actions::EntryAction
 
         def plan(repository, content_units, options = {})
           sync_capsule = options.fetch(:sync_capsule, true)
+
```
Comment (ianballou) [inline app/lib/actions/katello/repository/remove_content.rb:10]: You can remove this newline

**Review by ianballou — COMMENTED** [review]
Works fine for me! Just one comment.

**Review by ianballou — APPROVED** [review]
Looks fine now! Just need to wait for Jenkins.

**Pull request merged**